### DISCUSSION TITLE
[@types/async-retry] Broken types

### DISCUSSION THREAD

**Discussion by Morriz** [DefinitelyTyped/DefinitelyTyped#60193] [Issues with a @types package]
Hi, I think commit 6f77b38 broke async-retry. We get build errors now:
```
#12 38.96   Object literal may only specify known properties, and 'maxTimeout' does not exist in type 'WaitTillAvailableOptions'.
[547](https://github.com/redkubes/otomi-core/runs/6303461374?check_suite_focus=true#step:6:547)
#12 38.96 src/common/k8s.ts(197,5): error TS2322: Type '{ retries: number; maxTimeout: number; }' is not assignable to type 'Options'.
[548](https://github.com/redkubes/otomi-core/runs/6303461374?check_suite_focus=true#step:6:548)
#12 38.96   Object literal may only specify known properties, and 'retries' does not exist in type 'Options'.
```
But those are part of the official published interface: https://github.com/vercel/async-retry#readme

[a comment by an automated account is not shown]

**Comment by ghost** [comment]
Me too.

**Comment by BendingBender** [comment]
Just updated tests to test for all config properties inherited from `node-retry`: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/61820.

Everything seems to work fine, see tests in PR. Additionally, I've tested it with the setup [in this gist](https://gist.github.com/BendingBender/3eb57c65bcba341919b6ae69bff00d41). Can't reproduce your problem.

**Comment by ThomasFindlay** [comment]
For anyone who runs into this issue. You can fix it by installing @types/retry in addition to @types/async-retry as a dependency.

**Reply by HappyButter** [reply]
Works! Thanks!

**Reply by michaelk-redis** [reply]
Thank you! this helped me.
